tools/demo/index.html

```diff
@@ -6,9 +6,9 @@
         <title>PixiJS Filters Demo</title>
         <meta name='viewport' content='width=device-width, initial-scale=1.0'>
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
-        <link rel="stylesheet" href="index.css" />
         <script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.6.5/dat.gui.min.js"></script>
         <script src="https://pixijs.download/dev/pixi.min.js"></script>
+        <link rel="stylesheet" href="index.css" />
```

Comment:
Why this change?

Reply:
if `index.css` before `dat.gui.min.js` , some css in `index.css` will be ignored.  ( in fast network or use local dat.gui.min.js).